reference/conanfile/methods.rst

```diff
@@ -23,6 +23,7 @@ Methods
    methods/package_id
    methods/package_info
    methods/requirements
+   methods/run
```

Comment:
I think this is different, as this is not a recipe method. This section is to document the methods that a recipe should write, not methods that it can call like this one. Not sure either where/how to document it, but it is a bit confusing imo.

Reply:
I agree with this. I couldn't find a better place and after confirming with @czoido, I've moved it to the same place they have in Conan 1's docs: Another subsection in the `conanfile.py` one. Let me know what you think! :)